wm-paper/rfc/index.md

```diff
@@ -0,0 +1,91 @@
+# Stable UI
+
+After a lot of learnings, we can be proud that we have figured out what Watermelon will look like for the short and medium term. We have a stable UI that we can use to build the extension.
+
+## What is the stable UI?
+
+For this, we thought of all the possible integrations that will come in the future, and the broad categories of information that we will need to display. We also thought about the different ways to display this information, and the different ways to interact with it.
+
+Here are some examples:
+
+- Code Storage
+  - GitHub
+  - GitLab
+  - BitBucket
+- Ticketing
+  - Jira
+  - Trello
+  - Asana
+  - GitHub Projects
+  - Linear
+  - Notion
+- Communication
+  - Slack
+  - Discord
+  - Teams
+  - Zoom
+  - Google Meet
+- Documentation
```

Comment:
I'm starting to think documentation systems are out of scope at the moment. However we can choose from a myriad of directions to expand our product, so in the end it's about our opinion. 

Regardless, my opinion about documentation systems is the following:
- They're highly fragmented. Adding one doesn't increase our addressable market significantly. 
- The value prop yes, but not as much as a messaging system. Has potential though. 
- We should double down on this idea of "easily find passive documentation". It's what has given us an initial level of success.

Reply:
Yes, they are not the first to be proposed, only examples.